plugins/meta/vrf/vrf.go

```diff
@@ -130,6 +139,21 @@ CONTINUE:
 		}
 	}
 
+	// Apply all saved routes for the interface that was moved to the VRF
+	for _, route := range routes {
+		r := route
+		// Modify original table to vrf one,
+		// equivalent of 'ip route add <address> table <int>'.
+		r.Table = int(vrf.Table)
+		err = netlink.RouteAdd(&r)
```

Comment:
you can use netlink.RouteReplace instead (ip route replace)

Reply:
Hmm what do we get by using RouteReplace? I think if the route is there, we maybe not suppose to replace it...

Reply:
Make sense, thanks for the explanation. I will address it shortly.